hw/xbox/nv2a/gl/gloffscreen_wgl.c

```diff
@@ -43,29 +43,30 @@
  */
 
 struct GloMain {
-    HINSTANCE             hInstance;
-    HDC                   hDC;
-    HWND                  hWnd; /* Our hidden window */
-    HGLRC                 hContext;
+    HINSTANCE hInstance;
+    HDC hDC;
+    HWND hWnd; /* Our hidden window */
+    HGLRC hContext;
 };
 
 struct GloMain glo;
-int glo_inited = 0;
+int glo_inited;
```

Comment:
Initialization removed here too.
This seems to be a recurring issue. Was this intended?

While we are at it, this could also become a `bool`, possibly `static bool` like our existing code in GLX.

Reply:
See above. Agree to change to `bool` type.